test/controllers/AdditionalMetadataClosureStatusControllerSpec.scala

```diff
@@ -346,25 +356,19 @@ class AdditionalMetadataClosureStatusControllerSpec extends FrontEndTestHelper {
         |                                You must provide the following information, as they are mandatory for closure.
         |                            </span>""".stripMargin
     )
-    closureStatusPage must include(
-      """<ul class="govuk-list govuk-list--bullet govuk-list--spaced">
-        |                                <li>
-        |                                    FOI decision asserted, this is the date of the Advisory Council approval
-        |                                </li>
-        |                                <li>
-        |                                    Closure start date, this is the date the record starts
-        |                                </li>
-        |                                <li>
-        |                                    Closure period
-        |                                </li>
-        |                                <li>
-        |                                    FOI exemption code
-        |                                </li>
-        |                                <li>
-        |                                    Title public
-        |                                </li>
-        |                            </ul>""".stripMargin
-    )
+    closureStatusPage must include("""
+        |                            <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+        |XX
```

Comment:
Whats the `XX` for, is it to ensure the correct spacing is applied on the HTML fragment? If so, could be named something a bit more descriptive?